autoscale/src/main/java/com/sequenceiq/periscope/init/DefaultSubscriptionService.java

```diff
@@ -19,7 +19,7 @@
 
     private static final String DEFAULT_CLIENT_ID = "default";
 
-    @Value("${cb.default.subscription.address:}")
+    @Value("${cb.notification.endpoint:}")
     private String defaultSubscriptionAddress;
 
```

Comment:
Defaulting to the same naming convention as in CB .

Reply:
This will also be initialized with pod specific environment value in dps-k8s repository as done in CB.